ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -12,15 +12,38 @@
 {%     if router_isis.net is defined %}
 | Net-ID | {{ router_isis.net }} |
 {%     endif %}
+{%     if router_isis.router_id is arista.avd.defined %}
+| Router-ID | {{ router_isis.router_id }} |
+{%     endif %}
 {%     if router_isis.is_type is defined %}
 | Type | {{ router_isis.is_type }} |
 {%     endif %}
-{%     if router_isis.address_family is defined %}
-| Address Family | {% for af in router_isis.address_family %}{{ af }} {% endfor %}|
-{%     endif %}
 {%     if router_isis.log_adjacency_changes is arista.avd.defined %}
 | Log Adjacency Changes | {{ router_isis.log_adjacency_changes }} |
 {%     endif %}
+{%     if router_isis.mpls_ldp_sync_default is arista.avd.defined(true) %}
+| MPLS LDP Sync Default | {{ router_isis.mpls_ldp_sync_default }} |
+{%     endif %}
+{%     if router_isis.timers.local_convergence is arista.avd.defined %}
+| Local Convergence Delay (ms) | {{ router_isis.timers.local_convergence.delay | arista.avd.default(10000) }} |
+{%     endif %}
+{%     if router_isis.advertise.passive_only is arista.avd.defined(true) %}
+| Advertise Passive-only | {{ router_isis.advertise.passive_only }} |
+{%     endif %}
+{%     if router_isis.address_family is defined %}
+{%         set row_af = router_isis.address_family %}
+{%     else %}
+{%         set row_af = [] %}
+{%         if router_isis.address_family_ipv4 is arista.avd.defined %}
+{%             do row_af.append("ipv4") %}
+{%         endif %}
+{%         if router_isis.address_family_ipv6 is arista.avd.defined %}
+{%             do row_af.append("ipv6") %}
+{%         endif %}
+{%     endif %}
+{%     if row_af is arista.avd.defined %}
+| Address Family | {% for af in row_af %}{{ af }} {% endfor %}|
+{%     endif %}
```

Comment:
Line 33 - 46 can be simplified into:
```jinja
{%     if router_isis.address_family_ipv4 is arista.avd.defined %}
| IPv4 Address-family | True |
{%         if router_isis.address_family_ipv4.maximum_paths is arista.avd.defined %}
| IPv4 maximum-paths | {{ router_isis.address_family_ipv4.maximum_paths }} |
{%         endif %}
{%     endif %}
{%     if router_isis.address_family_ipv6 is arista.avd.defined %}
| IPv6 Address-family | True |
{%         if router_isis.address_family_ipv6.maximum_paths is arista.avd.defined %}
| IPv6 maximum-paths | {{ router_isis.address_family_ipv6.maximum_paths }} |
{%         endif %}
{%     endif %}
```
Maybe move all the TI LFA options to a separate section in the ISIS documentation, since that could grow.

Alternative is to create a new table for ipv4 AF and another for ipv6 AF with all the settings.

Reply:
Refactored this to dedicated tables for v4 and v6 AF.